.github/workflows/build.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     steps:
     - name: checkout
       uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
so with fetch-depth: 0 we don't need to set VERSION in GitHub action and in the past Jenkins was setting this by reading git describe. right?

Reply:
Yes, as long as VERSION isn't already set, the makefile will use `git describe` to automatically detect the version [here](https://github.com/rook/rook/blob/master/build/makelib/common.mk#L74). To work properly it needs the full history.